js/README.md

```diff
@@ -0,0 +1,39 @@
+# Berty `js/`
+
+__Please, read the main [`README.md`](../README.md) file first.__
+
+This folder contains most of the NodeJs code needed to develop the mobile apps.
+
+## UI component development
+
+Storybook enables isolated development of UI component libraries. Examples and source code are in the [packages](./packages) subfolder.
+
+### General requirements for React Native
+
+* `yarn` to install Nodejs dependencies
+* Homebrew or package manager of choice
+
+#### Requirements for iOS
+
+* OS X
+* Xcode [TODO: version?]
+* Node & Watchman (`brew install node`, `brew install watchman`)
+* CocoaPods (`sudo gem install cocoapods` or follow official [sudo-less instructions](https://guides.cocoapods.org/using/getting-started.html#sudo-less-installation))
+
+#### Requirements for Android
+
+* [TODO]
```

Comment:
Do you want to list the requirements to build the app for android in this PR ?

Reply:
Happy to 👍 I'll re-request review when added

Reply:
They are added @clegirar !